spec/integration/message_compiler_spec.rb

```diff
@@ -316,6 +316,34 @@
 
         expect(msg).to eql('length must be within 3 - 4')
       end
+
+      it "handles 'num' and 'size' interchangeably as interpolated arguments" do
+        messages = Dry::Schema::Messages::YAML.build.merge(
```

Comment:
Would you mind running RuboCop on this file to make the indentation consistent?

Reply:
Haha, that was even a method which was there in a first place but unfortunately I guess it was introduced after Ruby 2.4 and some of CLI for older version were failing with NoMethodErrors :(
I changed it to the `each_with_object`, I guess it's a little bit less awkward